packages/webdriverio/src/middlewares.js

```diff
@@ -18,7 +18,7 @@ export const elementErrorHandler = (fn) => (commandName, commandFn) => {
              *  - elementId couldn't be fetched in the first place
              *  - command is not explicit wait command for existance or displayedness
              */
-            if (!this.elementId && !commandName.match(/(wait(Until|ForDisplayed|ForExist|ForEnabled)|isExisting|isDisplayed)/)) {
```

Comment:
Previously commands like 'waitUntilFoo', 'isExistingFoo', 'isDisplayedFoo' were skipping waitForExist call. 
So I decided to add two more shortcuts (waitFor, waitToBe) and document all this.